packages/components/src/primitives/block-quotation/style.scss

```diff
@@ -0,0 +1,19 @@
+.wp-block-quote {
```

Comment:
This is the style for the block, I don't think it should have been moved to the component.

Reply:
Yes, you are correct. I missed it...

Reply:
This file was never referenced so it only needs to be deleted.

Reply:
https://github.com/WordPress/gutenberg/pull/15845 fixes it.